sycl/test/basic_tests/image_accessor.cpp

```diff
@@ -0,0 +1,44 @@
+// RUN: %clang -std=c++11 %s -o %t1.out -lstdc++ -lOpenCL -lsycl
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
+// RUN: %clang -std=c++11 -fsycl %s -o %t2.out -lstdc++ -lOpenCL -lsycl
+// RUN: env SYCL_DEVICE_TYPE=HOST %t2.out
+// RUN: %CPU_RUN_PLACEHOLDER %t2.out
+// RUN: %GPU_RUN_PLACEHOLDER %t2.out
+// RUN: %ACC_RUN_PLACEHOLDER %t2.out
+//==------------------- d;sjfas;fjs;fkjsa;dfjask;ldfja;slfjas---------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+#include <CL/sycl.hpp>
+#include <iostream>
+
+int main() {
+  cl::sycl::float4 src[256];
+  for (int i = 0; i < 256; i++) {
+    src[i] = {1.0f, 2.0f, 3.0f, 4.0f};
+  }
+
+  try {
+    cl::sycl::image<2> srcImage(src, cl::sycl::image_channel_order::rgba,
+            cl::sycl::image_channel_type::fp32, cl::sycl::range<2>(16, 16));
+
+    cl::sycl::queue myQueue;
+
+    myQueue.submit([&](cl::sycl::handler& cgh) {
+      cl::sycl::accessor<cl::sycl::float4, 2, cl::sycl::access::mode::read,
+      cl::sycl::access::target::image> inPtr( srcImage, cgh);
+      auto outPtr = srcImage.get_access<cl::sycl::float4,
+                        cl::sycl::access::mode::write>(cgh);
+    });
+
+  } catch (cl::sycl::exception e) {
+    std::cout << "SYCL exception caught: " << e.what();
+    return -1;
```

Comment:
I'm not sure I understand the purpose of this test.
Mustn't it check that exception is thrown if device do not support images?

Reply:
I guess there is no point of making test like that because images feature is not implemented yet.